dojo/settings/settings.dist.py

```diff
@@ -784,6 +784,8 @@ def generate_url(scheme, double_slashes, user, password, host, port, path, param
     'Acunetix Scan': ['title', 'description'],
     'Trivy Scan': ['title', 'severity', 'cve', 'cwe'],
     'Snyk Scan': ['vuln_id_from_tool', 'file_path', 'component_name', 'component_version'],
+    'SpotBugs Scan': ['cwe', 'severity', 'file_path'],
```

Comment:
you didn't put the line? interesting.

Reply:
I'm not really confident with the method used by spotbugs for the kotlin analysis (the testings foming from here), It's why I didn't add line actually for the duplicate. 

I think if we have more testing, we could add it after. This  why I propose to start like this.